rxgroups/src/main/java/com/airbnb/rxgroups/ObservableGroup.java

```diff
@@ -186,14 +208,14 @@ public void call(ManagedObservable<?> managedObservable) {
     }
 
     /**
-     * Returns an existing {@link Observable} for the provided {@code tag}. Does not change the
-     * locked
-     * status of this {@link ObservableGroup}. If it is unlocked, and the Observable has already
-     * emitted events, they will be immediately delivered. If it is locked then no events will be
+     * Returns an existing {@link Observable} for the provided {@code tag}.
+     * <p> Does not change the locked status of this {@link ObservableGroup}.
+     * If it is unlocked, and the Observable has already emitted events,
+     * they will be immediately delivered. If it is locked then no events will be
      * delivered until it is unlocked.
      */
     public <T> Observable<T> observable(AutoResubscribingObserver<? super T> observer) {
-        return observable(observer, null);
+        return observable(observer, observer.getTag());
```

Comment:
Yeah, I thought of that too, but if a user calls `transform(AutoResubscribingObserver)` without a tag, I feel we already assume it is a mistake to register any more observables to that observer. 

I feel like it's a little bit of a tricky concept to explain the difference between `transform(AutoResubscribingObserver, observableTag)` and `transform(AutoResubscribingObserver)`.

And I try to be as transparent as possible in the docs when using the variants without the tag e.g.:

```
Convenience method for {@link #cancelAndRemove(AutoResubscribingObserver, String)}, with
     * {@code observableTag} of {@link AutoResubscribingObserver#getTag()}.
     * <p> Use when the {@code observer} is associated with only one {@link Observable}.
```

Which feels a bit more clear than saying we will store the tag with a null observableTag.